zcash_client_sqlite/CHANGELOG.md

```diff
@@ -25,13 +25,21 @@ and this library adheres to Rust's notion of
 - An `unstable` feature flag; this is added to parts of the API that may change
   in any release. It enables `zcash_client_backend`'s `unstable` feature flag.
 - New summary views that may be directly accessed in the sqlite database.
-  The structure of these views should be considered unstable; they may 
+  The structure of these views should be considered unstable; they may
   be replaced by accessors provided by the data access API at some point
   in the future:
   - `v_transactions`
   - `v_tx_received`
   - `v_tx_sent`
 - `zcash_client_sqlite::wallet::init::WalletMigrationError`
+- A filesystem-backed `BlockSource` implementation
+  `zcash_client_sqlite::FsBlockDb`. This block source expects blocks to be
+  stored on disk in individual files named following the pattern
+  `<blockmeta_root>/blocks/compactblock-<blockheight>-<blockhash>`. A SQLite
```

Comment:
This is very minor and can be ignored.  However I had an Opinion to share.

I've gone back and forth over the years about singular versus plural for group names (either directories or java packages).  I think the singular is probably better because not all words pluralize, not all things should be pluralized, and pluralization goes against the underlying type which has a singular name.  It becomes more obvious when looking at a whole lot of java packages than when adding just a single directory.

E.g. consider Java packages with:
 - co.electriccoin.zcash.models - For typesafe model objects
 - co.electriccoin.zcash.db - For a single database, but which has multiple helper classes and thus justifies a singular package.

From that example, I think making models singular enhances the naming consistency and avoids introducing an additional complex naming rule (we only pluralize when it contains multiple items of the same type) by sticking with a simple naming rule (we always use singular).

Again, this is very minor and can be ignored as it has no impact on correctness.